src/services/bunker_cases/abnormal_cl_rebase.py

```diff
@@ -229,17 +225,70 @@ def _get_withdrawn_from_vault_between(self, prev_blockstamp: ReferenceBlockStamp
         return vault_withdrawals
 
     def _get_eth_distributed_events(self, from_block: BlockNumber, to_block: BlockNumber) -> list[EventData]:
+        """Get ETHDistributed events between blocks"""
         return self.w3.lido_contracts.lido.events.ETHDistributed.get_logs(
             fromBlock=from_block,
             toBlock=to_block,
         )
 
+    def _get_last_report_blockstamp(self, blockstamp: ReferenceBlockStamp) -> ReferenceBlockStamp:
+        """Get blockstamp of last report"""
+        last_report_ref_slot = self.w3.lido_contracts.get_accounting_last_processing_ref_slot(blockstamp)
+        return self._get_ref_blockstamp(needed=last_report_ref_slot, finalized=blockstamp.slot_number)
+
+    def _get_ref_blockstamp(self, needed: SlotNumber, finalized: SlotNumber) -> ReferenceBlockStamp:
+        """Get blockstamp of needed slot"""
+        return get_first_non_missed_slot(
+            self.w3.cc,
+            needed,
+            ref_epoch=EpochNumber(needed // self.c_conf.slots_per_epoch),
+            last_finalized_slot_number=finalized,
+        )
+
+    @staticmethod
+    def calculate_validators_count_diff_in_gwei(curr_validators: list[Validator], prev_validators: list[Validator]):
+        """Handle 32 ETH balances of freshly baked validators, who was activated between epochs"""
+        validators_diff_in_gwei = (len(curr_validators) - len(prev_validators)) * MAX_EFFECTIVE_BALANCE
```

Comment:
Let's add a comment about why it's safe, smth like:
Lido validators are counted by public keys that the protocol deposited with 32 ether, so we can safely count the differences in the number of validators when they occur by the size of the deposit. Any predeposits to Lido keys will not be counted until the key is deposited through the protocol and goes into the used state

Reply:
fixed